Spatial Data Type related Manuals Update (#478)
ALTIBASE/Documents: Manuals/Altibase_7.3/eng/Spatial SQL Reference.md

The input can be any type of spatial object that can be described using WKT. If the syntax of the input WKT is not valid, this function outputs an error.

> Users can set SRID during the creation of this object. Without any specific setting, the SRID of the created object is 0.

Reviewer: setting이 좀 어색한거 같아요.
geometry객체를 생성할때 보단 "이 함수를 이용할때"로 바꾸는게 나아보입니다.
-> Users can specify SRID when using this function. If not specified SRID, the SRID of the generated geometry object is 0.  또는 "If SRID is not specified"로 하셔도 좋을 것 같습니다.

Author: Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0
으로 수정하겠습니다~

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.
```

This function returns NULL if the value of the WKT argument is NULL.

> Users can set SRID during the creation of this object. Without any specific setting, the SRID of the created object is 0.

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

This function returns NULL if the value of the WKT argument is NULL.

> Users can set SRID during the creation of this object. Without any specific setting, the SRID of the created object is 0.

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

> Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.
```

This function returns NULL if the value of the WKT argument is NULL.

> Users can set SRID during the creation of this object. Without any specific setting, the SRID of the created object is 0.

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

> Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.
```

This function returns NULL if the value of the WKT argument is NULL.

> Users can set SRID during the creation of this object. Without any specific setting, the SRID of the created object is 0.

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

> Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.
```

 This function returns NULL if the value of the WKT argument is NULL.

> Users can set SRID during the creation of this object. Without any specific setting, the SRID of the created object is 0.

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

> Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.
```

This function returns NULL if the value of the WKT argument is NULL.

> Users can set SRID during the creation of this object. Without any specific setting, the SRID of the created object is 0.

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

This function returns NULL if the value of the WKT argument is NULL.

> Users can set SRID during the creation of this object. Without any specific setting, the SRID of the created object is 0.

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

> Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.```

Author: ```suggestion
Users can specify SRID when using this function. If SRID is not specified, the SRID of the generated geometry object is 0.
```

```

# 4. Spatial Data Migration

> This section describes how to migrate spatial data between Altibase products or other vendor database products.

Reviewer: 여기도 product보단 database 만 적는게 낫겠어요~

> This section describes how to migrate spatial data between Altibase products or other vendor database products.

Author: ```suggestion
This section describes how to migrate spatial data between Altibase servers or Altibase and the other vendor database.
```

### Spatial Data Migration Between Altibase Products

> 

Reviewer: 이거 국문의 C안에 맞춰서 재 작성해야 할것 같네요~

> Altibase Meta version 8.8.1 introduced a change in the spatial data storage format from WKB to EWKB. This alteration divides the scenarios for spatial data migration among Altibase products into four categories.

Reviewer: 공간데이터 저장 포멧이 변경되었다 라는 말이 먼저 나오는게 좋을 것 같아요. 그리고 "Altibase 제품간" 보다는 "Altibase 데이터베이스 간"이 더 맞는 표현일꺼 같아요. 
4가지 시나리오로 나뉜다. 라는 문장이영문으로 바꾸니깐,,먼가 어색한거 같아서,,
"공간 데이터를 마이그레이션 하기 위해서는 먼저 아래의 시나리오와 변경사항을 확인하라." 라는게 더 나을것 같아서,
아래와 같이 작성,,
The spatial data storage format was changed from WKB to EWKB in Altibase Meta version 8.8.1. For the spatial data migration among Altibase databases, first check the below scenarios and changes.
